beam-migrate/Database/Beam/Migrate/Simple.hs

```diff
@@ -338,15 +339,25 @@ runSimpleMigration :: MonadBeam be m
 runSimpleMigration runner hdl =
   runner hdl . mapM_ runNoReturn
 
+-- | Given a function to convert a command to a 'String', produce a script that
+-- will execute the given migration steps. Usually, the function you provide
+-- eventually calls 'displaySyntax' to render the command.
+backendMigrationStepsScript :: BeamSqlBackend be
```

Comment:
Can you mention `backendMigrationScript` in the docstring for `backendMigrationStepsScript`, and vice-versa? This way, users can more easily navigate the documentation